admin/src/js/controllers/display-languages.js

```diff
@@ -1,4 +1,5 @@
-const _ = require('underscore');
+const _ = require('lodash/core');
+_.uniq = require('lodash/uniq');
```

Comment:
Is this a recommended approach for accessing uniq? I think I'd prefer it be assigned to a new const, eg:

```suggestion
const uniq = require('lodash/uniq');
```

For one thing, I assume that this augmentation be stored on the `lodash/core` module meaning `_.uniq` is available everywhere. If that's the desired result then we should do it in the index file rather than in a specific controller.

Reply:
I was just doing it to preserve as much unchanged code as possible. 

I tried doing in the index file, but for some reason it didn't work. Then I assigned it as an Angular constant and had components use the constant. That did work but I hated it so rolled back. 

I can try assigning in the index file and see what happens, maybe I had fixed the browserify duplication thing and it will work this time around.